java/hibernate/src/main/resources/config.properties

```diff
@@ -0,0 +1,3 @@
+
+server.port = 8080
+db.schema = "ysql_hibernate"
```

Comment:
Lets use the default (`public`) schema so that it's more consistent with the other examples.
Can use `ysql_hibernate` as the database though.